element_in_unknown_length_array.java

```diff
@@ -0,0 +1,40 @@
+// Time Complexity : O(logn) to find high and O(logn) to find element in binary search
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : 
+// Any problem you faced while coding this : Yes, Incrementing low pointer didn't come to my mind, Also My initial code was to handle array out of bounds exception
+//But leetcode handles that by return INTEGER.MAX_VALUE which later was used in while loop for finding high
+
+
+// Your code here along with comments explaining your approach
+
+class Solution{
+	
+	public int search(ArrayReader reader, int target){
+		int high=1
```

Comment:
Please try to solve this code or reach out to the TAs or the professor.

Reply:
Hi, I solved this, Can you check the latest commit?

Reply:
Hi, I solved this, could you please check the latest commit for this?